### DISCUSSION TITLE
Sidebar expands when hovering too easy

### DISCUSSION THREAD

**Discussion by Fotispel** [zen-browser/desktop#1502] [General]
I often encounter an issue where the sidebar expands unintentionally when I try to click the browser’s back button. When my cursor moves slightly to the left, it can accidentally hover over the "+" button or the Workplace icon (if enabled), which triggers the sidebar to open.
This behavior is quite frequent for me and can be frustrating. A possible solution could be to add a dedicated button, similar to the Arc browser, that allows the user to expand the sidebar manually, reducing these accidental interactions.

Has anyone else experienced this issue?

**Comment by KarimElsayad247** [comment]
> a dedicated button, similar to the Arc browser, that allows the user to expand the sidebar manually,

This already exists! there should be a button that looks like two arrows.

![image](https://github.com/user-attachments/assets/2ceadf7e-6a18-4f9c-8441-9b1e478ac447)
